File: leetcode/448.py
from typing import List

class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:

        for n in nums:
            idx = abs(n)-1
            if nums[idx]>0:
                nums[idx] = -1 * nums[idx]
        
        res = []
        for idx, n in enumerate(nums):
            if n >0:
                res.append(idx+1)
        return res



nums = [1]

sol =Solution()
ans = sol.findDisappearedNumbers(nums)
print(ans)

File: leetcode/arrays/448.go
package main

import "fmt"

func abs(num int) int {
	if num <0{
		return -1*num
	} 
	return num
}

func findDisappearedNumbers(nums []int) []int {

	ans := []int{}
	for _, ele :=  range nums{
		val := abs(ele)
		if nums[val-1] > 0{
			nums[val-1] *= -1
		}
	}

	for idx, ele := range nums{
		if ele > 0{
			ans = append(ans, idx+1)
		}
	}
	
	return ans
}

func main() {

	nums := []int{1,1,3,4}

	ans := findDisappearedNumbers(nums)

	fmt.Println(ans)
}